doc/locale/ja/LC_MESSAGES/news/15.po

```diff
@@ -44,9 +53,6 @@ msgstr ""
 msgid "Added support for MySQL 8.0.42"
 msgstr "MraiaDB 8.0.42をサポート"
```

Comment:
```suggestion
msgstr "MySQL 8.0.42をサポート"
```

Reply:
I will fix the typo in another PRs because there are still some typos.